### ISSUE TITLE
Subscriptions created using `checkout` aren't cancellable by `cancel`

### ISSUE THREAD

**Issue by thenbe** [tierrun/tier#249]
### 1. Create sub with `tier.subscribe()`, then call `tier.lookupPhase()` yields:
```json
		"lookupPhase": {
			"effective": "2023-02-16T03:46:52.000Z",
			"features": ["feature:unit@plan:perunit@1"],
			"plans": ["plan:perunit@1"]
		},
```

### 2. Create sub with `tier.checkout()`, then call `tier.lookupPhase()` yields:
```json
		"lookupPhase": {
			"effective": "2023-02-16T03:44:46.000Z",
			"features": ["feature:unit@plan:perunit@1"],
			"fragments": ["feature:unit@plan:perunit@1"]
		}
```

Both are using the same `features` arg. The second one is not cancellable. Calling `tier.cancel()` returns an error:

```
TierError: Internal Server Error
    at Tier.apiPost (file:///home/username/projects/try-tier/node_modules/.pnpm/tier@5.1.1/node_modules/tier/dist/mjs/client.js:521:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Tier.cancel (file:///home/username/projects/try-tier/node_modules/.pnpm/tier@5.1.1/node_modules/tier/dist/mjs/client.js:601:16)
    at async Object.cancel (file:///home/username/projects/try-tier/node_modules/.pnpm/tier@5.1.1/node_modules/tier/dist/mjs/index.js:107:12)
    at async unsubscribe (/src/routes/[lang=lang]/organizations/[organizationId]/billing/+page.server.ts:52:5)
    at async Module.handle_action_json_request (/@fs/home/username/projects/try-tier/node_modules/.pnpm/@sveltejs+kit@1.5.6_svelte@3.55.1+vite@4.1.1/node_modules/@sveltejs/kit/src/runtim
tions.js:57:16)
    at async resolve (/@fs/home/username/projects/try-tier/node_modules/.pnpm/@sveltejs+kit@1.5.6_svelte@3.55.1+vite@4.1.1/node_modules/@sveltejs/kit/src/runtime/server/respond.js:371:17

    at async handleAuth (/src/hooks/auth.ts:54:20)
    at async handleLocale (/src/hooks/locale.ts:34:20)
    at async handleSentry (/src/hooks/sentry.ts:34:20) {
  path: '/v1/subscribe',
  requestData: { org: 'org:5f2488a5-bb7d-4f20-946b-b7a7b41e7ce4', phases: [ {} ] },
  status: 500,
  code: 'internal_error',
  responseData: {
    status: 500,
    code: 'internal_error',
    message: 'Internal Server Error'
  },
  cause: undefined
}
```

---

Side question:

Why does one return `fragments`, while the other returns `plans`, and where can I read more about the difference between the two?

### Steps to reproduce

```json
{
	// "$schema": "https://www.tier.run/docs/pricing.schema.json",
	"plans": {
		"plan:perunit@1": {
			"title": "myplan",
			"interval": "@monthly",
			"features": {
				"feature:unit": {
					"aggregate": "perpetual",
					"tiers": [{ "price": 500, "base": 20000 }]
				}
			}
		}
	}
}
```

### OS

Linux

### OS version

Ubuntu 22.04

### Tier version

0.7.1, 0.8.0

**Comment by bmizerany** [comment]
Thank you for the ticket!

This is likely due to a known issue with checkout which is that it assumes one wants a new subscription per session. We have plans to make this less confusing in Tier, but for now one needs to check that there is not already a subscription before calling checkout. This feature is still Alpha and will receive plenty of polish as we move further ahead with it.

For more context. Please see https://github.com/tierrun/tier/issues/224

Regarding the 500: Tier defaults to "panic" if it sees two subscriptions it thinks it created by accident. This should not happens without checkout atm.

**Comment by bmizerany** [comment]
Regarding fragments: We do need to add docs around this, but I hope this is helpful in the meantime:

I can't deep link right to it, but in this comment please scroll to where I touch on the subject of fragments.

https://github.com/tierrun/tier/issues/120#issuecomment-1319407957

**Comment by thenbe** [comment]
Thank you for the links, I'll have to read up on those.

> for now one needs to check that there is not already a subscription before calling checkout

This is happening on brand new orgs/subs. If their first subscription was through `checkout`, then it can't be cancelled.

**Comment by bmizerany** [comment]
Confirmed. I'm working on getting a fix and a release out.

**Comment by bmizerany** [comment]
@thenbe Fixed and available in v0.8.1, which was just released. Please let me know if you have any more trouble.
